src/server/snapshot.cc

```diff
@@ -68,13 +67,18 @@ bool SliceSnapshot::IsSnaphotInProgress() {
 void SliceSnapshot::Start(bool stream_journal, const Cancellation* cll, SnapshotFlush allow_flush) {
   DCHECK(!snapshot_fb_.IsJoinable());
 
-  auto db_cb = absl::bind_front(&SliceSnapshot::OnDbChange, this);
+  auto db_cb = [this](DbIndex db_index, const DbSlice::ChangeReq& req) {
```

Comment:
why did you do this changes?

Reply:
`bind_front` is implemented via chain of functions that add to stack trace usage hundreds of bytes (in debug mode). In general I do not see why bind_front is needed nowdays if we have lambdas.